src/infra/cron/cronCmdRepo_test.go

```diff
@@ -10,45 +10,57 @@ import (
 
 func TestCronCmdRepo(t *testing.T) {
 	testHelpers.LoadEnvVars()
+	cronCmdRepo, err := NewCronCmdRepo()
+	if err != nil {
+		t.Errorf("UnexpectedError: %v", err)
+	}
 
 	t.Run("CreateCron", func(t *testing.T) {
-		schedule := valueObject.NewCronSchedulePanic("* * * * *")
-		command, _ := valueObject.NewUnixCommand("echo \"cronTest\" >> crontab_log.txt")
-		comment := valueObject.NewCronCommentPanic("Test cron job")
+		schedule, err := valueObject.NewCronSchedule("* * * * *")
+		if err != nil {
+			t.Errorf("UnexpectedError: %v", err)
+		}
```

Comment:
You can discard errors on VO creation of unit tests. Won't mention that on every case, so please keep an eye for all repetitions.

Reply:
Done.